config/widgets/settings/generalOptionsTab.json

```diff
@@ -185,11 +183,22 @@
 			],
 			"callback": "availableCreaturesAsDwellingChanged"
 		},
+		{
+			"name": "compactTownCreatureInfoLabel",
+			"type" : "verticalLayout",
+			"customType" : "labelDescription",
+			"position": {"x": 45, "y": 391},
+			"items" : [
+				{
+					"text": "vcmi.otherOptions.compactTownCreatureInfo.hover",
+				}
+			]
+		},
 		{
 			"name": "compactTownCreatureInfoCheckbox",
 			"type": "checkbox",
 			"help": "vcmi.otherOptions.compactTownCreatureInfo",
-			"position": {"x": 10, "y": 383},
+			"position": {"x": 10, "y": 389},
```

Comment:
Why change? Checkboxes are 30px apart:
- 323; creatureGrowthAsDwelling
- 353: availableCreaturesAsDwelling
- so compactTownCreatureInfo should be at 383, as before

Reply:
I have added a line divider to separate the 2 toggle options. Figured out a line is about 6 :)

Reply:
It is actually 0 :)
Lines don't affect layout right now